Makefile

```diff
@@ -20,6 +20,9 @@ osbs-build:
 	# hack/build.sh
 	./tmp/build/build.sh
 
+unit:
+	go test -v ./pkg/...
```

Comment:
@kevinrizza My bad, I forgot to mention that we do not commit the generated mock files into git. So we would need to generate the mock files before the unit tests can be run.  `make generate-mocks` will generate the mocks for you in the right place.

Reply:
Please open a Jira bug to track this.